components/compact-table-accordion-group/src/index.js

```diff
@@ -2,11 +2,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'react-emotion';
 import { Collapse } from 'react-collapse';
+import {
```

Comment:
why not a single-liner?
would be a bit tidier :grin:

Reply:
Yea I will tidy this actually, it crossed my mind as I was doing it, but then I forgot to update it.